distributor-api-v1/operations.md

```diff
@@ -768,34 +768,60 @@ Gives a pricing information for the given configuration.
 
 |  | Property | Type | Description |
 | :--- | :--- | :--- | :--- |
-| `HotelId` | string | required | Unique identifier of hotel |
-| `Client` | string | required | Identification of the client as described in [Authorization](https://mews-systems.gitbook.io/distributor-guide/distributor-api-v1/authorization) |
-| `Session` | number | required | Session number |
-| `LanguageCode` | string | required | Language code |
-| `CultureCode` | string | required | Culture Code |
+| `Client` | string | required | Identification of the client as described in [Authorization](https://mews-systems.gitbook.io/distributor-guide/distributor-api-v1/authorization). |
+| `HotelId` | string | required | Unique identifier of hotel. |
+| `Session` | number | required | Session number. |
```

Comment:
why are the dots added if it's not a sentence?

Reply:
After we decide, please create a follow up task to fix the dots everywhere to make it consistent.

Reply:
Will create an issue for it.